.github/kokoro/install.sh

```diff
@@ -48,8 +48,12 @@ echo "----------------------------------------"
 	rm -rf build
 	du -ah install
 	export GIT_HASH=$(git rev-parse --short HEAD)
-	tar cfJv symbiflow-arch-defs-install-${GIT_HASH}.tar.xz -C install bin share environment.yml
+	tar cfJv symbiflow-arch-defs-install-${GIT_HASH}.tar.xz -C install bin share/symbiflow/techmaps share/symbiflow/scripts environment.yml
 	tar cfJv symbiflow-arch-defs-benchmarks-${GIT_HASH}.tar.xz -C install benchmarks
+	for device in $(ls install/share/symbiflow/arch)
+	do
+		tar cfJv symbiflow-$device.tar.xz -C install share/symbiflow/arch/$device
```

Comment:
Can you explain why this is `symbiflow-<device>` rather than `symbiflow-<git hash>-<device>`?

Reply:
One reason would be that it would make easier to generate the latest link name in the GH actions script (https://github.com/SymbiFlow/symbiflow-arch-defs/blob/a0dfd328c974930671f6fb9e4a47f3959547c611/.github/check-status/check-status.py#L33), but I can fix this to be `symbiflow-arch-defs-<device>-<git hash>` and make the latest link generation capable of extracting the device name

Reply:
Done